minecraft_servers/nextlevel-crimelife/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+  "server_name": "nextlevel-crimelife",
+  "nice_name": "Nextlevel-Crimelife",
+  "direct_ip": "nextlevel-crimelife.net",
+  "server_wildcards": [
+    "%.nextlevel-crimelife.net"

```

Comment:
Doesn't look like a real wildcard
![image](https://github.com/LabyMod/server-media/assets/37186532/37a6f477-aaf0-4ae9-b3d6-5559e4645df9)

Reply:
now it should work